compose/service.py

```diff
@@ -1120,6 +1118,12 @@ def build(self, no_cache=False, pull=False, force_rm=False, memory=None, build_a
 
         return image_id
 
+    def get_cache_from(self, build_opts):
+        cache_from = build_opts.get('cache_from', None)
+        if cache_from:
```

Comment:
Should this be `cache_from is not None`? (honest question, I'm not sure)

Reply:
I fixed it